react-course/video-24/demo.html

```diff
@@ -0,0 +1,46 @@
+<body>
+    <div id="root"></div>
+    <script src="https://unpkg.com/react@16.12.0/umd/react.development.js"></script>
+    <script src="https://unpkg.com/react-dom@16.12.0/umd/react-dom.development.js"></script>
+    <script src="https://unpkg.com/@babel/standalone@7.8.3/babel.js"></script>
+    <script src="https://unpkg.com/react-error-boundary@1.2.5/dist/umd/react-error-boundary.js"></script>
+    <script type="text/babel">
+        const allItems = [
+            { id: 'a', value: 'apple' },
+            { id: 'o', value: 'orange' },
+            { id: 'g', value: 'grape' },
+            { id: 'p', value: 'pear' },
+        ]
+
+        function App() {
+            const [items, setItems] = React.useState(allItems)
+
+            function addItem() {
+                setItems([...items, allItems.find(i => !items.includes(i))])
```

Comment:
`.find` is quite new. let's discuss it in a session

Reply:
okay, because he didn't explain much about it.